### PR TITLE
Improve Issue Template.

### PR DESCRIPTION
**Pull request by vpn1997**
[ WIP ]

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -9,8 +32,5 @@ If you'd like to report a bug, please include the following information:
 - Command line used:
 - Steps to reproduce:
```
Comment (fristonio) [inline .github/ISSUE_TEMPLATE.md]: Steps to reproduce are there two times.

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -1,6 +1,29 @@
-Thanks for taking the time to report a bug in the Origami project, or propose a new feature! Before creating a new issue, please do a quick search to check that it hasn't already been filed on the [issue tracker](https://github.com/Cloud-CV/Origami/issues)._
+Thanks for taking the time to report a bug in the Origami project, or propose a new feature! Before creating a new issue, please do a quick search to check that it hasn't already been filed on the [issue tracker](https://github.com/Cloud-CV/Origami/issues).
+For more information, see the [CONTRIBUTING Guide](https://github.com/Cloud-CV/Origami/blob/master/.github/CONTRIBUTING.md) 
+
+## Description
+
+[Description of the bug or feature]</br>
```
Comment (fristonio) [inline .github/ISSUE_TEMPLATE.md]: Without a <br/> here would be better. We should include html as less as possible.

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -1,6 +1,29 @@
-Thanks for taking the time to report a bug in the Origami project, or propose a new feature! Before creating a new issue, please do a quick search to check that it hasn't already been filed on the [issue tracker](https://github.com/Cloud-CV/Origami/issues)._
+Thanks for taking the time to report a bug in the Origami project, or propose a new feature! Before creating a new issue, please do a quick search to check that it hasn't already been filed on the [issue tracker](https://github.com/Cloud-CV/Origami/issues).
+For more information, see the [CONTRIBUTING Guide](https://github.com/Cloud-CV/Origami/blob/master/.github/CONTRIBUTING.md)
```
Comment (fristonio) [inline .github/ISSUE_TEMPLATE.md]: I was thinking of having something like this too https://github.com/angular/angular.js/blob/master/DEVELOPERS.md

Comment (vpn1997) [reply .github/ISSUE_TEMPLATE.md]: Are you referring to the contributing guide ?

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (vpn1997) [reply .github/ISSUE_TEMPLATE.md]: :+1:

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (vpn1997) [reply .github/ISSUE_TEMPLATE.md]: Yup.Thanks.

File: .github/ISSUE_TEMPLATE.md (hunk 3 above)
Comment (fristonio) [reply .github/ISSUE_TEMPLATE.md]: No, I was thinking that we should have a guide like the above for new developer to maintain a continuant style throughout origami. Take for instance https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-coding-rules

Comment (vpn1997) [reply .github/ISSUE_TEMPLATE.md]: We should make these changes in the contributing guide .

Comment (fristonio) [reply .github/ISSUE_TEMPLATE.md]: Contributing guide would be fine, I just want this information to be present somewhere so as to enforce a continuant style throughout the project.

**Comment by uttu357** [comment]
Once a PR is finalized request a review on that PR.

**Comment by vpn1997** [comment]
@uttu @AvaisP  please review this PR.

**Pull request merged**